### PR TITLE
feat-added Questions Page

### PR DESCRIPTION
**Pull request by prathmeshjagtap**
Added Question Page for Quiz

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]

File: styles/question.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	background-color: #1c1b1b;
+	margin: 1.5rem 0;
+	padding: calc(3 * var(--padding-px-l));
+	border-radius: var(--border-radius-m);
+	text-align: center;
+	font-size: 1.2rem;
+	cursor: pointer;
+}
+.options li:hover,
+.quiz__options__container a:hover {
```
Comment (Hardiegogo) [inline styles/question.css]: Try to use classes instead of using descendant selectors unless it is necessary.

**Review by Hardiegogo — COMMENTED** [review]
look at some changes suggested below.

**Review by spraveenofficial — APPROVED** [review]
**Good to merge.**

File: styles/question.css (hunk 1 above)
Comment (prathmeshjagtap) [reply styles/question.css]: yes did the required  changes

**Pull request merged**